docs/reference/modules/network/threading.asciidoc

```diff
@@ -0,0 +1,106 @@
+[[modules-network-threading-model]]
+==== Networking threading model
+
+This section describes some technical details about the threading model used by
+the networking subsystem in {es}, which may be important for advanced users to
+diagnose certain problems in a cluster.
+
+{es} nodes communicate over a collection of TCP channels that together form a
+transport connection. {es} clients communicate with the cluster over HTTP,
+which also uses one or more TCP channels. Each of these TCP channels is owned
+by exactly one of the `transport_worker` threads in the node. The owning thread
+of a channel is chosen when the channel is opened, and cannot change.
+
+Each `transport_worker` thread has sole responsibility for sending and
+receiving data over the channels it owns. One of the `transport_worker` threads
+is also responsible for accepting new incoming transport connections, and one
+is responsible for accepting new HTTP connections.
```

Comment:
```suggestion
is also responsible for accepting new incoming transport connections. Another
`transport_worker` thread is responsible for accepting new HTTP connections.
```

Reply:
I know what you mean but it might be the same thread in both cases. "Another" implies that isn't the case.